src/Settings/Flavours/QuickCross.hs

```diff
@@ -0,0 +1,23 @@
+module Settings.Flavours.QuickCross (quickCrossFlavour) where
+
+import Flavour
+import Expression
+import Oracles.Flag
+import {-# SOURCE #-} Settings.Default
+
+quickCrossFlavour :: Flavour
+quickCrossFlavour = defaultFlavour
```

Comment:
Both `quickCrossFlavour` and `quickCrossArgs` are mostly copied from `quickFlavour`. I wonder if we can/should reduce the duplication. Maybe it doesn't worth it.

@izgzhen What do you think?

Reply:
I think we should keep this duplication, since it looks more clear what is implicated by `cross` or `quick-cross`, even if part of them overlaps.